### ISSUE TITLE
App crashes trying to select habit frequency

### ISSUE THREAD

**Issue by lucanomax** [iSoron/uhabits#926]
## Pre-submission checklist
- [x] I am submitting a bug report, not a feature request.
- [x] I am running the latest version of Loop Habit Tracker.
- [x] I have have searched for similar issues, but did not find any matches.

## Description
Crash when trying to select days in new habit "yes or no"

## Steps to reproduce
1. Create new habit
2. *Yes or no* type
3. Tap on *Frequency*
4. App crashes

## Logs
```
time: 1621750457389
msg: java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
stacktrace: java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.Collections$SingletonList.get(Collections.java:4872)
	at org.isoron.uhabits.activities.common.dialogs.FrequencyPickerDialog.addBeforeAfterText(FrequencyPickerDialog.kt:113)
	at org.isoron.uhabits.activities.common.dialogs.FrequencyPickerDialog.onCreateDialog(FrequencyPickerDialog.kt:55)
	at androidx.fragment.app.DialogFragment.onGetLayoutInflater(DialogFragment.java:380)
	at androidx.fragment.app.Fragment.performGetLayoutInflater(Fragment.java:1412)
	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:881)
	at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:1238)
	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:1303)
	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:439)
	at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManagerImpl.java:2079)
	at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:1869)
	at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:1824)
	at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:1727)
	at androidx.fragment.app.FragmentManagerImpl$2.run(FragmentManagerImpl.java:150)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7397)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)
```

## System information
 - Phone: Redmi 5+ (vince)
 - Phone Operating System: Android 10 (AeX 7.3)
 - App version: 2.0.1

**Comment by iSoron** [comment]
Thank you for the bug report, @lucanomax. I was able to reproduce the bug by setting the system language to Italian. It seems like one of the strings was not translated correctly. It should be fixed in Loop 2.0.2.

**Comment by iSoron** [comment]
Fixed in branch `hotfix/2.0.2`
